elastic/kibana
# [Metrics UI] Add anomalies table to flyout

## Summary

Fixes: #88649 sans the new anomaly indicator. 

## Test & Screenshots
<details>
<summary>Anomaly Flyout Jobs tab</summary>
No Jobs:
<img width="1277" alt="Screen Shot 2021-02-22 at 10 25 28 AM" src="https://user-images.githubusercontent.com/1676003/108729267-57e0e500-74f8-11eb-813f-0a55f7e6ab6c.png">

- The callout should function as before, showing which jobs you have enabled.
- Some text and styles were updated (see figma)

With Jobs:
<img width="1276" alt="Screen Shot 2021-03-03 at 3 19 51 PM" src="https://user-images.githubusercontent.com/1676003/109868005-2624f800-7c35-11eb-9d63-7e32a673f18f.png">

</details>

<details>
<summary>Anomalies tab table</summary>

- Anomalies tab should show table of anomalies and results should reflect `anomalyThreshold` value in Settings
- Ability to sort by `Time` or `Severity` columns
- Choose between `hosts` or `k8s` anomalies using the dropdown

<img width="920" alt="Screen Shot 2021-02-17 at 1 38 20 PM" src="https://user-images.githubusercontent.com/1676003/108251482-73797380-7125-11eb-8a5a-5fb24c1ed9e0.png">
</details>

<details>
<summary>Anomalies tab table search</summary>

- search should call api to query for the job id

search Kubernetes
<img width="919" alt="Screen Shot 2021-02-17 at 1 44 09 PM" src="https://user-images.githubusercontent.com/1676003/108252088-3bbefb80-7126-11eb-9e0d-036d8450ca02.png">

search Hosts
<img width="920" alt="Screen Shot 2021-02-17 at 1 42 29 PM" src="https://user-images.githubusercontent.com/1676003/108251979-17631f00-7126-11eb-8aff-564273b40345.png">

</details>

<details>
<summary>Anomalies table no results</summary>

- Table should show "Loading anomalies" when loading or "No anomalies" messaging
<img width="918" alt="Screen Shot 2021-02-17 at 1 45 38 PM" src="https://user-images.githubusercontent.com/1676003/108252324-804a9700-7126-11eb-879c-07f23b809624.png">
</summary>
</details>

<details>
<summary>Anomaly table row actions menu</summary>

- "Show in inventory" should take the user to the waffle map to the correct Host/k8 with the relative metric type, time, and query filtered by the partition field name and value.  If the user did not select a partition field, then there is no filter.  If you open the timeline the anomaly should be present.  
- "Open in anomaly explorer" should take the user to the anomaly explorer app filtered by job id
- "Create Alert" should open the create alert flyout

<img width="916" alt="Screen Shot 2021-02-17 at 1 49 07 PM" src="https://user-images.githubusercontent.com/1676003/108252672-ef27f000-7126-11eb-9e2a-dd8fd3f5b9eb.png">
</details>

### Checklist
